packages/front-end/components/DatePicker/index.tsx

```diff
@@ -133,6 +135,7 @@ export default function DatePicker({
                 style={{ width: inputWidth, minHeight: 38, overflow: "clip" }}
               >
                 <Field
+                  id={id ?? ""}
```

Comment:
default should be `undefined`
```suggestion
                  id={id}
```

Reply:
It was throwing a linting error without